src/renderer/ssf-api.ts

```diff
@@ -332,6 +332,17 @@ export class SSFApi {
         }
     }
 
+    /**
+     * Cancel a screen capture in progress
+     *
+     * @param screenSnippetCallback {function}
+     */
+    public closeScreenSnippet(): void {
```

Comment:
I don't think we need to expose this API. 
Closing the snippet should be handled on the SDA side and an exception to be thrown

Reply:
This is in preparation for https://perzoinc.atlassian.net/browse/SDA-1284 which will require the frontend to issue a "closeScreenSnippet "command when the user leaves the "Create wall post" dialog.

Reply:
ok 👍